preprocess.yaml

```diff
@@ -140,11 +146,9 @@ targets:
   preprocess:
     depends:
       - SB_county_boundaries_USA_json
-      - SB_county_boundaries_zoom_json
       - SB_county_boundaries_mobile_json
+      - SB_county_boundaries_zoom_json
       - SB_state_boundaries_USA_json
-      - SB_state_boundaries_zoom_json
       - SB_state_boundaries_mobile_json
-      - SB_all_boundaries_USA_json
-      - SB_all_boundaries_zoom_json
```

Comment:
:+1: for cleaning up the outputs. i thought for a while that the _all_ files would become useful, but they haven't turned out that way.